prueba/src/Primos/Identificador.java

```diff
@@ -0,0 +1,21 @@
+package Primos;
+
+public class Identificador {
+
+	public double num;
+	 public void identificador()
+	 {
+		 int contador=0;
+		 for(int i=1;i<num+1;i++)
+		 {
+			 if((num/i) % 1==0)
```

Comment:
I cannot understand the intention of this line by reading it. 
Code should reveal the intention of the programmer just by reading it :P something like:
```java
private bool numberMeetsProperty(num) {
  return (num/i) % 1 == 0;
}

if (numberMeetsProperty(num)) {
  contador++;
}
```
    
Maybe the name of the function is not the  best, but it gives more intention of what the programmer wanted to do.